src/snowflake/snowpark/column.py

```diff
@@ -255,11 +271,16 @@ def __init__(
 
             # some repetition here, but _expression logic will be eliminated eventually
             if self._ast is None:
-                self._ast = proto.SpColumnExpr()
                 if expr1 == "*":
-                    self._ast.sp_column_sql_expr.sql = "*"
+                    self._ast = Column._create_ast(
```

Comment:
As soon as https://github.com/snowflakedb/snowpark-python/pull/1701 is in , can we add tests for this?